apps/server/src/modules/user-import/mapper/role-name.mapper.spec.ts

```diff
@@ -1,4 +1,4 @@
-import { IImportUserRoleName } from '@shared/domain/entity';
+import { ImportUserRoleName } from '@modules/user-import/entity';
```

Comment:
The mapper here is in the "user-import" context, perhaps it is better to use a relative import e.g. '../entity', like what you had done in `role-name.mapper`